doc/basics.md

```diff
@@ -1016,6 +1017,102 @@ If your documents are large (e.g., larger than a megabyte), then the `iterate_ma
 
 See [iterate_many.md](iterate_many.md) for detailed information and design.
 
+
+
+Parsing Numbers Inside Strings
+------------------------------
+
+The simdjson library supports parsing valid numbers inside strings through three methods: `get_double_in_string`, `get_int64_in_string` and  `get_uint64_in_string`. However, it is important to
```

Comment:
I suggest adding a bit more motivation. Something like this:

"""
Though the JSON specification allows for number and string values, many engineers choose to integrate the numbers inside strings. E.g., they prefer `{"a":"1.9"}` to`{"a":1.9}`.

"""

Reply:
I would add the motivation at the very beginning. This is really not obvious unless you have seen it before.